<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>624</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>INFLUXDB数据库配置界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../influxdb/image.qrc">
    <normaloff>:/influxDb.png</normaloff>:/influxDb.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="Tab1">
      <attribute name="title">
       <string>服务器配置</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7" rowstretch="9,1" columnstretch="3,7">
       <item row="0" column="0">
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QSpinBox" name="index1">
            <property name="minimum">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="Label_2">
            <property name="text">
             <string>名称:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="Label">
            <property name="text">
             <string>索引:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="username"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="Label_5">
            <property name="text">
             <string>用户名:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="enable1">
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="port">
            <property name="styleSheet">
             <string notr="true">默认:1883</string>
            </property>
            <property name="text">
             <string>1883</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="Label_4">
            <property name="text">
             <string>备注:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="password"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="Label_6">
            <property name="text">
             <string>密码:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="Label_8">
            <property name="text">
             <string>端口号:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="desc1"/>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="address"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="Label_3">
            <property name="text">
             <string>使能:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="name1"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="Label_7">
            <property name="text">
             <string>地址:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSpinBox" name="timeOut">
            <property name="minimum">
             <number>1000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>超时时间:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTableView" name="tableView1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="connectTest">
            <property name="text">
             <string>连接测试</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearService">
            <property name="text">
             <string>清除数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveService">
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Tab2">
      <attribute name="title">
       <string>主题配置</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6" rowstretch="9,1" columnstretch="3,7">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="1">
           <widget class="QComboBox" name="service"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="name2"/>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="frequency">
            <property name="minimum">
             <number>1000</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>服务器:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="Label_15">
            <property name="text">
             <string>索引:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="Label_18">
            <property name="text">
             <string>备注:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="index2">
            <property name="minimum">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="enable2">
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>采样频率:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="Label_16">
            <property name="text">
             <string>名称:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="desc2"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="Label_17">
            <property name="text">
             <string>使能:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="topic"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>主题:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="publish">
            <property name="text">
             <string>发布测试</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearTopic">
            <property name="text">
             <string>清除数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveTopic">
            <property name="text">
             <string>保存</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTableView" name="tableView2"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../influxdb/image.qrc"/>
 </resources>
 <connections/>
</ui>
